php/Block.php

```diff
@@ -24,6 +25,20 @@ class Block {
 	 */
 	protected $plugin;
 
+	/**
+	 * Holds the meta key for pre-rendered block.
+	 *
+	 * @var string
+	 */
+	const META_KEY = 'xwp-site-counts';
```

Comment:
I've noted that this should be prefixed with an underscore to make it private. 🤦‍♂️